app/serializers/search/reaction_serializer.rb

```diff
@@ -0,0 +1,32 @@
+module Search
+  class ReactionSerializer
+    include FastJsonapi::ObjectSerializer
+
+    attributes :id, :category, :status, :user_id
+
+    attribute :reactable do |reaction|
```

Comment:
I went ahead and made this compatible for comment and article reactions. To begin we will only be using this for articles but I wanted it to be a seamless transition when we were ready to expand its use.